### ISSUE TITLE
[BUG] Strokes: Unhandled Exception Caught: Exception caught

### ISSUE THREAD

**Issue by dsguffey** [CGCookie/retopoflow#851]
![RetopoFlow Strokes error](https://user-images.githubusercontent.com/65681495/85963353-350b2a80-b983-11ea-9afa-5119f24375a9.JPG)

**Comment by vxlcoder** [comment]
Thanks for the report, @dsguffey.

I'm a little confused, because this error should not occur (there are checks in place for this).  Did you get this version of RF from the Blender Market, from GitHub, or somewhere else?

**Comment by dsguffey** [comment]
Github. I got version 3-beta-2 under releases, but I had to manually
install it due to Blender constantly wanting to put it in 2.82 addons
instead of 2.83, which is what I'm using. It's entirely possible that
there's something wrong either with my installation of Blender or
RetopoFlow. I'll probably end up deleting Blender and reinstalling it at
some point just to see if that helps with some of the issues I've been
having. If my bug reports aren't helpful, let me know and I'll stop sending
them.
